app/routes/_public.undercut.tsx

```
@@ -1,339 +1,336 @@
-              labelTitle="Retainer Name"
-              inputTag="Name"
-              name="retainerName"
-              toolTip="The name of the retainer that is selling your item"
+              type="number"
+              labelTitle="Ignore stacks larger than:"
+              inputTag="Stack Size"
+              value={config.ignore_undercuts_with_quantity_over}
+              onChange={(e) =>
+                setConfig((prev) => ({
+                  ...prev,
+                  ignore_undercuts_with_quantity_over: parseInt(
```

Biome (1.9.4)

[error] Use Number.parseInt instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseInt instead.


(lint/style/useNumberNamespace)